src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -34,7 +34,7 @@ class InputOption
 
     /**
      * @param string                        $name        The option name
-     * @param string|array                  $shortcut    The shortcuts, can be null, a string of shortcuts delimited by | or an array of shortcuts
+     * @param string|array|null             $shortcut    The shortcuts, can be null, a string of shortcuts delimited by | or an array of shortcuts
```

Comment:
getShortcut also needs a patch  :joy: (string|null)

Reply:
maybe better to cast to string there? What do you think?

Reply:
that's a BC break :)